performance-tests/performance.test.js

```diff
@@ -5,6 +5,11 @@ const { getByText, getByLabelText, waitFor } = require('@testing-library/dom')
 const { default: userEvent } = require('@testing-library/user-event')
 const { startApp } = require('./test-app')
 
+// number of times to run the test
+const NUMBER_OF_RUNS = 50
+// the number of tests to toss (slowest and fastest)
+const BUFFER = 10
```

Comment:
including a buffer now, which is the number of slowest (and fastest) runs that we'll toss when doing calculations. In reality, this means that we are only getting stats for 30 runs (50 runs, - 10 slowest, - 10 fastest), but these should be more consistent overall.